.github/workflows/update_locales.yml

```diff
@@ -0,0 +1,57 @@
+name: Update Locales
+
+on:
+    workflow_dispatch:
+    push:
+        tags:
+            - '*'
+
+permissions:
+    contents: write
+    pull-requests: write
+
+jobs:
+    sync_localise:
+        uses: ./.github/workflows/sync_tags_localise.yml 
+        secrets: inherit  # pass all secrets
+
+    update-locales:
+        needs: [sync_localise]  # require sync_localise to pass before deploy runs
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v4
+
+            - uses: actions/setup-python@v5
+              with:
+                python-version: '3.10'
```

Comment:
If there are no incompatibilities, maybe we can use python v3.11 or v3.12 so end-of-life arrives 1 or 2 years later and we can reduce maintainability efforts. Otherwise, 3.10 should also work fine
![image](https://github.com/user-attachments/assets/1af346b5-62d0-4840-a886-67fb048ab3f4)

Reply:
Resolved at 8fde9ed